ghost/admin/lib/koenig-editor/addon/components/koenig-card-embed.js

```diff
@@ -208,6 +210,9 @@ export default class KoenigCardEmbed extends Component {
                 return;
             }
             this.set('hasError', true);
+            if (err.message === 'Ajax authorization failed') {
+                this.set('isUnauthorized', true);
+            }
```

Comment:
Rather than checking the message text it should be possible to use the `isUnauthorizedError()` function from the [ember-ajax addon](https://github.com/ember-cli/ember-ajax#error-handling) we use, which should be a little more robust as it uses the response status rather than message. Example here https://github.com/TryGhost/Ghost/blob/main/ghost/admin/app/routes/application.js#L92

Reply:
Awesome thanks for the tip!